Sources/UI/Views/ReactionsView.swift

```diff
@@ -36,6 +36,8 @@ final class ReactionsView: UIView {
         return view
     }()
     
+    public var preferredEmojiOrder = ["👍", "❤️", "😂", "😲", "😔", "😠"]
```

Comment:
IMHO this should live under `ChatViewController`, close to `emojiReactionTypes` so it's easily configurable (by users using different reactions)
You can follow the same structure: `preferredEmojiOrder` as `var`, `defaultPreferredEmojiOrder` as `open` so it's used by overriding classes